### PR TITLE
Added Link to memes.market in the reddit comment for !firm command

### PR DESCRIPTION
**Pull request by Caribosaurus**
This PR changes the bot answer to the !firm command. It simply adds a link to the memes.market firm page.

### CODE REVIEW

**Comment by Caribosaurus** [comment]
@thecsw

**Review by thecsw — APPROVED** [review]
Hey there! Everything seems fine to me. Approved! Thank you for your support!

**Comment by Caribosaurus** [comment]
Just realized the formatting isn’t right.

**Comment by thecsw** [comment]
Where?

**Comment by Caribosaurus** [comment]
message.py 417 462  (** in the url)

File: src/message.py [hunk 1]
```diff
@@ -379,7 +414,7 @@ def modify_pay_to_post(balance):
 """
 
 firm_other_org = """
-Firm: **%FIRM_NAME%**
+Firm: [**%FIRM_NAME%**](https://meme.market/firm.html?firm=**%FIRM_ID%**)
```
Comment (thecsw) [inline src/message.py]: Yeah, just delete the astericks and we're fine

File: src/message.py [hunk 2]
```diff
@@ -408,20 +443,23 @@ def modify_pay_to_post(balance):
...
+        replace("%EXECS%", execs). \
+        replace("%ASSOCS%", assocs). \
+        replace("%TRADERS%", traders). \
+        replace("%BALANCE%", "{:,}".format(firm.balance)). \
         replace("%LEVEL%", str(firm.rank + 1))
 
+
 firm_self_org = """
-Firm: **%FIRM_NAME%**
+Firm: [**%FIRM_NAME%**](https://meme.market/firm.html?firm=**%FIRM_ID%**)
```
Comment (thecsw) [inline src/message.py]: And here

Comment (Caribosaurus) [reply src/message.py]: Will do!

**Comment by Caribosaurus** [comment]
good to go

**Comment by thecsw** [comment]
Approved. Merging and deploying

**Pull request merged**